pyaedt/edb.py

```diff
@@ -1356,36 +1356,123 @@ def solve_siwave(self):
         return True
 
     @pyaedt_function_handler()
-    def add_design_variable(self, variable_name, variable_value):
+    def add_design_variable(self, variable_name, variable_value, is_parameter=False):
         """Add a design variable.
 
         Parameters
         ----------
         variable_name : str
-            Name of the variable.
+            Name of the variable. To be added as a project variable, the name must begin with ``$``.
         variable_value : str, float
             Value of the variable with units.
+        is_parameter : bool, optional
+            The default is ``True`` to add the variable as a parameter default.
+            If ``False``, the variable is added as a local variable.
```

Comment:
```suggestion
            Whether to add the variable as a parameter default. The default is ``True``. 
            If ``False``, the variable is added as a local variable.
```

Reply:
@jeremich41 I edited again only to follow what we set as the standard format style for Boolean descriptions.